recipes/seq-hasher/meta.yaml

```diff
@@ -6,16 +6,19 @@ package:
 
 build:
   number: 0
+  script:
+    - cargo-bundle-licenses --format yaml --output THIRDPARTY.yml
```

Comment:
as you have a `build.sh`, adding this `script:` directive will make the build not use that script I believe. Move the `cargo-bundle-licenses --format yaml --output THIRDPARTY.yml` into the `build.sh`. Take a look at https://github.com/conda-forge/ssubmit-feedstock/tree/main/recipe for an example